docs/api.rst

```diff
@@ -419,6 +421,15 @@ Multi-step Update
    :members:
 
 
+Split Complex
```

Comment:
Perhaps we can add some more detail to help users use this? (NB. If we move this to experimental, we should add a section for that here)